configs/common/Options.py

```diff
@@ -38,13 +38,13 @@
 
 import argparse
 
+from common import ObjectList
```

Comment:
There is something weird happening here. It seems like isort isn't detecting this as a local import. It might be possible to fix this by doing something like `from . import ObjectList` instead. IIRC, there was a reason for including the full path here and it had to do with Python 2.7 compatibility.

There are similar issues in multiple example configuration files where local imports get confused.

I think this can be solved using relative imports. That should tell isort that it is a local import and it would avoid the ugly import path fixup.